types/react-embed-gist/react-embed-gist-tests.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
```

Comment:
you probably don't need this to be a .tsx file

Reply:
I'm pretty sure you can only include JSX in a .tsx file, when I changed it to a .ts file the test failed but the test runs fine when it's a .tsx file, so I think this should work.

Reply:
oh! alright!